static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy.js

```diff
@@ -13,13 +13,15 @@ const messageTextV2 =
     'This year, much of what we hold dear has been threatened - democracy, civility, truth. This administration is establishing new norms of behaviour. Truth is being chased away. With your help we can continue to put it center stage. As we prepare for 2020, we’re asking our readers to help us raise $1.5 million.';
 const ctaText = 'Support The Guardian';
 
+const tickerHeaderControl = 'Help us reach our year-end goal';
+
 export const contributionsBannerUsEoy: AcquisitionsABTest = {
```

Comment:
can this banner be deleted?

Reply:
I'm leaving it for now as no decisions have been made about how long the current ones will run for

Reply:
it will be (hopefully) obvious to remove it when the campaign is over